Dockerfile

```diff
@@ -1,5 +1,5 @@
 # That's the only place where you're supposed to specify version of Trivy.
-ARG TRIVY_VERSION=0.20.1
+ARG TRIVY_VERSION=0.22.0
```

Comment:
There is one more place to bump version of Trivy in https://github.com/aquasecurity/harbor-scanner-trivy/blob/main/test/component/component_test.go#L25 (That's the reason why component tests are currently failing.)